test/unit/spec/v2/view-builder/internals/FormInputFactory_spec.js

```diff
@@ -38,13 +38,15 @@ describe('v2/view-builder/internals/FormInputFactory', function () {
         factorType: 'password',
         iconClassName: 'mfa-okta-password',
         description: 'Choose a password for your account',
+        buttonDataSeAttr: '',
```

Comment:
why is this empty?

Reply:
This test is for the old LEA implementation that has `factorType` instead of `authenticatorType`. This implementation also uses the same `getAuthenticatorData` in `AuthenticatorUtil` - https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/internals/FormInputFactory.js#L12